core/trino-main/src/test/java/io/trino/execution/TestPlannerWarnings.java

```diff
@@ -101,7 +100,7 @@ public static void assertPlannerWarnings(LocalQueryRunner queryRunner, @Language
                     createPlan(queryRunner, transactionSession, sql, warningCollector, rules.get());
                 }
                 else {
-                    queryRunner.createPlan(transactionSession, sql, CREATED, false, warningCollector);
+                    queryRunner.createPlan(transactionSession, sql, OPTIMIZED, false, warningCollector);
```

Comment:
where does this one come from?

Reply:
`CREATED` did not run `DetermineTableScanNodePartitioning`